frontend/app/src/components/accounts/blockchain/ChainSelect.vue

```diff
@@ -3,46 +3,15 @@ import { Blockchain } from '@rotki/common/lib/blockchain';
 import ChainDisplay from '@/components/accounts/blockchain/ChainDisplay.vue';
 import { Module } from '@/types/modules';
 
-interface SupportedChain {
-  symbol: Blockchain;
-  name: string;
-  icon?: string;
-}
-
-const chains: SupportedChain[] = [
-  {
-    symbol: Blockchain.ETH,
-    name: 'Ethereum'
-  },
-  {
-    symbol: Blockchain.ETH2,
-    name: 'Beacon chain validator'
-  },
-  {
-    symbol: Blockchain.BTC,
-    name: 'Bitcoin'
-  },
-  {
-    symbol: Blockchain.BCH,
-    name: 'Bitcoin Cash'
-  },
-  {
-    symbol: Blockchain.OPTIMISM,
-    name: 'Optimism',
-    icon: './assets/images/chains/optimism.svg'
-  },
-  {
-    symbol: Blockchain.KSM,
-    name: 'Kusama'
-  },
-  {
-    symbol: Blockchain.DOT,
-    name: 'Polkadot'
-  },
-  {
-    symbol: Blockchain.AVAX,
-    name: 'Avalanche'
-  }
+const chains = [
```

Comment:
we don't really need this anymore, we could get this values from the supported chains now right?